.github/workflows/coding-standards.yml

```diff
@@ -15,3 +15,7 @@ jobs:
     uses: "doctrine/.github/.github/workflows/coding-standards.yml@1.4.1"
     with:
       composer-options: "--prefer-dist --prefer-stable"
+
+  lint-composer:
+    name: "Lint Composer"
+    uses: "phansys/doctrine.github/.github/workflows/composer-lint.yml@composer_lint"
```

Comment:
This should not reference your fork

Reply:
It's temporary, for test purposes, in order to proove that the upstream PR works before merging it.

Reply:
Let's convert this to a draft in the meantime.